### PR TITLE
Clarify refreshing package list and installing deb

### PR DESCRIPTION
**Pull request by karth295**
1) Move `apt-get update` into the code section. It was a bit hidden in the description before.
2) Remove brackets around package to install, as it prevents simply copying and pasting the command.

#### Checklist
- [x] documentation is changed or added (if applicable)
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/handlebars/installation.handlebars [hunk 1]
```diff
@@ -334,9 +334,12 @@
       <pre><code class="highlighted">sudo apt-get install -y software-properties-common</code></pre>
 
       <p>Then repeat the first command.</p>
-      <p>3. Refresh your package list with <tt>sudo apt-get update</tt> and then install your chosen AdoptOpenJDK package. For example, to
+      <p>3. Refresh your package list with and then install your chosen AdoptOpenJDK package. For example, to
```
Comment (karianna) [inline src/handlebars/installation.handlebars:337]: Grammar doesn't make sense here?  Refresh your package list with and...

**Review by karianna — CHANGES_REQUESTED** [review]

**Comment by gdams** [comment]
@aahlenst do you think we need to clarify here? To me, the original wording seems pretty clear

**Comment by aahlenst** [comment]
Well, `apt-get update` was moved to the code sample. Maybe it's more visible that way. But I don't know if it really helps. The problem is the entire page.

What we really need is a nice assistant that helps you generate the right configuration and commands but that's out of scope here.

**Comment by gdams** [comment]
okay, let's close this PR for now and we can revisit

File: src/handlebars/installation.handlebars (hunk 1 above)
Comment (karth295) [reply src/handlebars/installation.handlebars:337]: Oops, yeah my PR should have deleted the word `with` as well.

**Pull request closed without merging**